edgeless_systemtests/src/lib.rs

```diff
@@ -0,0 +1,288 @@
+            }
+        };
+
+        // Create 10 workflows
+        let mut workflow_ids = vec![];
+        cleanup();
+        for workflow_i in 0..num_workflows {
+            let res = client
+                .start(edgeless_api::workflow_instance::SpawnWorkflowRequest {
+                    workflow_functions: vec![
+                        edgeless_api::workflow_instance::WorkflowFunction {
+                            name: "f1".to_string(),
+                            function_class_specification: fixture_spec(),
+                            output_mapping: std::collections::HashMap::from([
+                                ("out1".to_string(), "f2".to_string()),
+                                ("out2".to_string(), "f3".to_string()),
+                                ("log".to_string(), "log".to_string()),
+                            ]),
+                            annotations: std::collections::HashMap::from([("init-payload".to_string(), "8".to_string())]),
+                        },
+                        edgeless_api::workflow_instance::WorkflowFunction {
+                            name: "f2".to_string(),
+                            function_class_specification: fixture_spec(),
+                            output_mapping: std::collections::HashMap::from([("log".to_string(), "log".to_string())]),
+                            annotations: std::collections::HashMap::new(),
+                        },
+                        edgeless_api::workflow_instance::WorkflowFunction {
+                            name: "f3".to_string(),
+                            function_class_specification: fixture_spec(),
+                            output_mapping: std::collections::HashMap::from([("log".to_string(), "log".to_string())]),
+                            annotations: std::collections::HashMap::new(),
+                        },
+                    ],
+                    workflow_resources: vec![edgeless_api::workflow_instance::WorkflowResource {
+                        name: "log".to_string(),
+                        class_type: "file-log".to_string(),
+                        output_mapping: std::collections::HashMap::new(),
+                        configurations: std::collections::HashMap::from([("filename".to_string(), removeme_filename(workflow_i))]),
+                    }],
+                    annotations: std::collections::HashMap::new(),
+                })
+                .await;
+            workflow_ids.push(match res {
+                Ok(response) => match &response {
+                    edgeless_api::workflow_instance::SpawnWorkflowResponse::ResponseError(err) => {
+                        panic!("workflow rejected: {}", err)
+                    }
+                    edgeless_api::workflow_instance::SpawnWorkflowResponse::WorkflowInstance(val) => {
+                        assert_eq!(4, val.domain_mapping.len());
+                        for i in 0..4 {
```

Comment:
Would try to not depend on the order here. Easiest way around that would be to compare sets.